azureLogIngestion/opentelemetry/provider.ts

```diff
@@ -0,0 +1,10 @@
+import { Resource } from "@opentelemetry/resources"
+import { BasicTracerProvider } from "@opentelemetry/tracing"
+
+export class NRTracerProvider extends BasicTracerProvider {
+    resource: Resource
```

Comment:
For this class definition, the typescript hinting complains that `TS2610: 'resource' is defined as an accessor in class 'BasicTracerProvider', but is overridden here in 'NRTracerProvider' as an instance property.`

But if it tests and builds...we might be able to ignore that. (That's what led me down the path of trying to work with the private `_resource` property.

Reply:
Also, when I run the test, I'm getting that TS2610 error.

Reply:
I'm running `npm run build` without error. Is this some IDE you're running? That may be enabling additional linting.

Reply:
I'm running my builds also with `npm run build` in the terminal from within IntelliJ. I do get the same build errors from my default terminal, though. My test branch fails on build in github, though curiously this PR doesn't seem to have run that same build step.